_includes/scripts.html

```diff
@@ -41,46 +41,55 @@
     a.appendChild(r);
   })(window, document, "https://static.hotjar.com/c/hotjar-", ".js?sv=");
 </script>
+{% endif %}
+
+<!-- Intercom -->
 <script>
   window.intercomSettings = {
     app_id: "rzcjacgd"
   };
 </script>
-{% endif %}
-
-<!-- Intercom -->
 <script>
-  (function() {
+  (function () {
     var w = window;
     var ic = w.Intercom;
     if (typeof ic === "function") {
       ic("reattach_activator");
-      ic("update", intercomSettings);
+      ic("update", w.intercomSettings);
     } else {
       var d = document;
-      var i = function() {
+      var i = function () {
         i.c(arguments);
       };
       i.q = [];
-      i.c = function(args) {
+      i.c = function (args) {
         i.q.push(args);
       };
       w.Intercom = i;
-      function l() {
+      var l = function () {
```

Comment:
What's `var l` for? It doesn't appear to be used

Reply:
It is later called on line 78, 80 & 82. I copied the latest install script from Intercom.

Reply:
Ah, I see. Could this have a better name like "bootstrapIntercom" or something along those lines? I always get concerned with single character variable names :-/

Reply:
I'm leaving this as-is simply because this is the default script from Intercom's guides. If we want to change the variable name; we should do that for all the other single character variables contained in the script as well.